barretenberg/cpp/src/barretenberg/ultra_honk/decider_proving_key.cpp

```diff
@@ -321,7 +321,7 @@ void DeciderProvingKey_<Flavor>::move_structured_trace_overflow_to_overflow_bloc
             // ensures it can be read into by the previous gate but does not itself try to read into the next gate.
             for (auto& selector : block.get_gate_selectors()) {
                 BB_ASSERT_EQ(selector.empty(), false);
-                selector.back() = 0;
+                selector.set_back(0);
```

Comment:
will you please document `set_back` as it's used around the proving system "front facing" classes